### PR TITLE
#193: calculate average release hoc and commits size

### LINKED ISSUE #193: `average_release_hoc_size` into QoS
Let's add `average_release_hoc_size` metric to the `quality-of-service.rb` script. It should calculate the average number of hits-of-code in a published release. Also, let's add `average_release_commits_size` with the average number of commits in a release.

### PR DESCRIPTION
**Pull request by Yegorov**
Closes #193

### CODE REVIEW

**Comment by Yegorov** [comment]
@Suban05 please review. Test fail, because this https://github.com/zerocracy/fbe/pull/95 PR need merge before

**Review by Suban05 — CHANGES_REQUESTED** [review]
@Yegorov looks good, but see some the comment

File: judges/quality-of-service/quality-of-service.rb [hunk 1]
```diff
@@ -66,6 +66,27 @@
   diffs = (1..dates.size - 1).map { |i| dates[i] - dates[i - 1] }
   f.average_release_interval = diffs.empty? ? 0 : diffs.inject(&:+) / diffs.size
 
+  # Release hoc and commit size
+  repo_releses = {}
```
[the remark this answers is not in the document]
Comment (Yegorov) [reply judges/quality-of-service/quality-of-service.rb]: @Suban05 Yes, of course, thanks, fixed!

**Review by Suban05 — APPROVED** [review]
@Yegorov 👍

**Comment by Yegorov** [comment]
@yegor256 please check

**Comment by 0crat** [comment]
@Suban05 Great job on the review! You've earned +4 points: +4 base, -10 for limited comments, but +10 to recognize your effort. Your commitment shines through! 🌟 Current balance: [+250](https://zerocracy.github.io/judges-action/zerocracy-vitals.html). Keep up the good work!

**Comment by 0crat** [comment]
@Yegorov 🎉 Great work on your contribution! You've earned +10 points: +4 base, +9 for 196 hits-of-code, -4 for limited comments. [Keep up](https://www.yegor256.com/2018/03/06/speed-vs-quality.html) the momentum! Your balance is now [+170](https://zerocracy.github.io/judges-action/zerocracy-vitals.html). Looking forward to your next submission!

**Pull request merged**